public/static/docs/command-reference/remote/add.md

```diff
@@ -194,50 +194,12 @@ $ dvc remote add myremote "azure://"
 
 ### Click for Google Drive
 
-Since Google Drive has tight API usage quotas, creation and configuration of
-your own `Google Project` is required:
-
-1.  Log into the [Google Cloud Platform](https://console.developers.google.com)
-    account.
-2.  Create `New Project` or select available one.
-3.  Click `ENABLE APIS AND SERVICES` and search for `drive` to enable
-    `Google Drive API` from search results.
-4.  Navigate to
-    [All Credentials](https://console.developers.google.com/apis/credentials)
-    page and click `Create Credentials` to select `OAuth client ID`. It might
-    ask you to setup a product name on the consent screen.
-5.  Select `Other` for `Application type` and click `Create` to proceed with
-    default `Name`.
-6.  `client id` and `client secret` should be showed to you. Use them for
-    further DVC's configuration.
+> 💡 Before adding a Google Drive remote, be sure to check
+> [Setup Google Drive Remote](/doc/user-guide/setup-google-drive-remote) user
+> guide.
```

Comment:
Sounds good, I'm going to rephrase above to more compact thing:

```
Check out [Setup Google Drive Remote](/doc/user-guide/setup-google-drive-remote) guide to create, add, and configure Google Drive remotes.
```

Please let me know if this will require further adjustments, thanks

Reply:
Thanks, it's pretty good. I just changed it a little bit in e9ec763c.